introduction.md

```diff
@@ -0,0 +1 @@
+Hi, i'm pcassimans and learning to work with git effectively :-)
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.